coralproject/talk
# Suspect wordlist

## What does this PR do?

This PR adds the ability to manage a "suspect words" list. Words that are not outright banned, but will be highlighted for the mods in the queue. Words can be managed in a tag manager style, and I've updated the Banned Words component as well. I added a bunch of words, so if @okbel or @gabelula could take a look at the coral-admin translations, I would appreciate it.

## How do I test this PR?

- `npm install`
- delete the settings in your db and restart the app `db.settings.remove({})` since the wordlist has changed its shape.
- go to the Configure Page > Banned & Suspect Words
- type some naughty words and hit Enter or Tab to add them
- make a list of `naughty, words, separated, by, commas` and paste it in. They should be automagically parsed.
- click Save
- go back to the Moderation Queue and view comments with those words. They should be highlighted in yellow.
